pages/common/netserver.md

```diff
@@ -0,0 +1,21 @@
+# netserver
+
+> Server-side command for `netperf`, the benchmarking application that measures network throughput.
+> See also: `netperf` for the client-side command.
+> Homepage: <https://hewlettpackard.github.io/netperf/>.
```

Comment:
```suggestion
> More information: <https://manned.org/netserver.1>.
```

Reply:
Why use manned.org for one but the HP github.io page for the other command?

Reply:
> Why use manned.org for one but the HP github.io page for the other command?

In general, we go with the official documentation and use manned.org as I fallback, as stated in the style guide. In this case, I didn't found a official documentation for netserver, so I used manned.org